bool parking_or_out
float32 line_distance
float32 start_x
float32 start_y
float32 start_ori
float32 end_x
float32 end_y
float32 end_ori
